### PR TITLE
Add rapidcheck/20200131 recipe

### PR DESCRIPTION
**Pull request by madebr**
Specify library name and version:  **rapidcheck/~20200213~20200131**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/rapidcheck/all/conandata.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+sources:
+  "20200213":
```
Comment (uilianries) [inline recipes/rapidcheck/all/conandata.yml]:
```suggestion
  "20200131":
```

The date of that commit is January 31: https://github.com/emil-e/rapidcheck/commit/258d907da00a0855f92c963d8f76eef115531716

File: recipes/rapidcheck/all/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class RapidcheckConan(ConanFile):
+    name = "rapidcheck"
+    description = "QuickCheck clone for C++ with the goal of being simple to use with as little boilerplate as possible"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/emil-e/rapidcheck"
+    license = "BSD-2-clause"
```
Comment (uilianries) [inline recipes/rapidcheck/all/conanfile.py]:
```suggestion
    license = "BSD-2-Clause"
```

File: recipes/rapidcheck/all/patches/0002-cmake-fix-install.patch [hunk 3]
```diff
@@ -0,0 +1,11 @@
+--- CMakeLists.txt
```
Comment (uilianries) [inline recipes/rapidcheck/all/patches/0002-cmake-fix-install.patch]: This patch has been copied from #863 hahaha ctrl+c ctrl+v curse

File: recipes/rapidcheck/all/patches/0001-remove-deprecated-attribute.patch [hunk 4]
```diff
@@ -0,0 +1,11 @@
+--- a/qrencode.h
```
Comment (uilianries) [inline recipes/rapidcheck/all/patches/0001-remove-deprecated-attribute.patch]: ditto

File: recipes/rapidcheck/all/patches/0002-cmake-fix-install.patch (hunk 3 above)
Comment (madebr) [reply recipes/rapidcheck/all/patches/0002-cmake-fix-install.patch]: Haha, you have eagle eys! I'm sorry again! 😸

**Pull request merged**